tools/pkgchk/cmd/root.go

```diff
@@ -1,76 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-package cmd
```

Comment:
We should not remove this tool, I believe this should be accidental. Could we have this tool back? This tool is used in the pipeline of the track 1 SDK

Reply:
done